Mathlib/Topology/MetricSpace/Countable.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2025 Bryan Wang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Geoffrey Irving, Bryan Wang
+-/
+import Mathlib.Analysis.Real.Cardinality
+
+/-!
+# Properties of countable metric spaces.
+
+-/
+
+open Set
+
+universe u
+
+variable {γ : Type u} [MetricSpace γ]
+variable {s : Set γ}
```

Comment:
There's no need to handle universes manually here so let's let Lean take charge. Additionally let's declare `s` where it is used since it is used just once.
```suggestion
variable {γ : Type*} [MetricSpace γ]
```